3party/opening_hours/opening_hours_supported_features_tests/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(opening_hours_supported_features_tests)
+
+include_directories(${PROJECT_SOURCE_DIR} ${OMIM_ROOT}/3party/opening_hours)
+
+set(
+  SRC
+  opening_hours_supported_features_tests.cpp
+)
+
+add_executable(opening_hours_supported_features_tests ${SRC})
```

Comment:
Can we use omim_add_test? Also, it's better to use ```${PROJECT_NAME}``` instead of opening_hou...._tests here and below.